src/main/java/Duke.java

```diff
@@ -73,9 +96,39 @@ public static void addList() {
                 continue;
             }
             System.out.println("    ____________________________________________________________");
-            list.add(line);
+            System.out.println("     Got it. I've added this task: ");
             done.add(0);
-            System.out.println("     added: " + line);
+            String description;
+            String time;
+            if (line.startsWith("todo")) {
+                type.add("T");
+                description = line.substring(5);
+                Todo todo = new Todo(description);
+                list.add(description);
+                date.add("");
+                System.out.println("       " + todo);
+            } else if (line.startsWith("deadline")) {
+                type.add("D");
+                int seperate = line.indexOf("/by");
+                description = line.substring(9, seperate - 1);
+                time = line.substring(seperate + 4);
+                Deadline deadline = new Deadline(description, time);
+                list.add(description);
+                date.add(time);
+                System.out.println("       " + deadline);
+            } else if (line.startsWith("event")) {
+                type.add("E");
+                int seperate = line.indexOf("/at");
```

Comment:
Be careful in the indexOf("/at"). Sometimes the input might not be /at but just starting with /